pkg/deployment/inventory.go

```diff
@@ -172,6 +172,14 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 		"inventory": string(invData),
 	}
 	secretName := fmt.Sprintf("dataplanenodeset-%s", instance.Name)
+	labels := map[string]string{
+		"openstack.org/operator-name": "dataplane",
+		"nodeset":                     instance.Name,
```

Comment:
can this be openstackdataplanenodeset to match the labels we add elsewhere (ansibleEE)?